quality/management/commands/create_sample_ops.py

```
@@ -0,0 +1,157 @@
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from quality.models import OrdemProducao
```

Pylint (3.3.7)

[error] No name 'OrdemProducao' in module 'quality.models'

(E0611)